tempesta_fw/http_sess.c

```diff
@@ -1144,7 +1349,7 @@ tfw_http_sess_get_srv_conn(TfwMsg *msg)
 	read_lock(&sess->lock);
```

Comment:
This is the lock addressed for the optimization in https://github.com/tempesta-tech/tempesta/issues/685#issuecomment-289420714 : `__try_conn()` is lock-free and `tfw_http_sticky_sess_enabled()` is also only about checking a bit, so it seems we do not need the lock here.

Reply:
@ikoveshnikov any comment/fix on this? This is in the #685 comment, so it should be fixed in the PR to close the task.